vip_spec.xsd

```diff
@@ -200,6 +202,7 @@ reference elements for polling location
             <xs:sequence>
               <xs:element name="PartyID" type="xs:IDREF" minOccurs="0" />
               <xs:element name="PersonID" type="xs:IDREF" minOccurs="0" />
+              <!-- SURVEY: Should external identifiers be referenced or included inline? -->
```

Comment:
Personally, I think inline makes more sense.  I also think we should define an "ExternalIdentifiers" (or "ExternalIdentifierCollection") type since everywhere we want to allow external identifiers, it will always be a collection.